src/mocks/table.ts

```diff
@@ -0,0 +1,34 @@
+import { TableData } from '@prefecthq/prefect-design'
+import { MockFunction } from '@/services/Mocker'
+import { choice } from '@/utilities'
+
+export type MockTableDataTypes = 'noun' | 'number' | 'boolean' | 'dateString' | 'email' | 'runName'
+
+export type MockTableOverrides = {
+  keys?: string[],
+  types?: MockTableDataTypes[],
+  columnCount?: number,
+  rowCount?: number,
+}
+
+export const randomTable: MockFunction<TableData[], [Partial<MockTableOverrides>?]> = function(overrides = {}) {
```

Comment:
This is cool. I can see this being expanded to a randomRecord and randomList that can create a list of objects that all have the same type but who's properties could have values from any of the other mocks